public/img/svg/email-relay-touts.svg

```diff
@@ -0,0 +1 @@
+<svg id="b89430c4-19ab-4184-973e-72ca05c2a86f" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 293.25 160.29"><title>email-relay-touts</title><path d="M255.23,36.06,241.82,48.88,229.88,37.52l-1-.91-1.53,1.62L192.52,73.69a7.35,7.35,0,0,0,5.2,2.15H286a7.35,7.35,0,0,0,5.19-2.15l-34.3-35.9Z" transform="translate(-0.12 -0.02)" fill="#7542e5"/><path d="M228.92,36.67,192.47,2.21a7.78,7.78,0,0,0-2.11,5.13V68.6a7.36,7.36,0,0,0,2.16,5.2l34.87-35.46ZM292.83,5a7.48,7.48,0,0,0-2-3.13L255.12,35.91l1.62,1.62,34.47,36.16a7.36,7.36,0,0,0,2.16-5.2V7.22A7.38,7.38,0,0,0,292.83,5Z" transform="translate(-0.12 -0.02)" fill="#9059ff"/><path d="M228.92,36.67l1,.91,11.94,11.36,13.41-12.82L291,2a7.11,7.11,0,0,0-4.87-2H197.72a7.12,7.12,0,0,0-5.18,2.27Z" transform="translate(-0.12 -0.02)" fill="#c689ff"/><path d="M57.65,69.3,45.77,80.67,34.41,69.84,33,71.22l-31,31.51a6.52,6.52,0,0,0,4.61,1.91h78.2a6.52,6.52,0,0,0,4.61-1.91L59.06,70.8Z" transform="translate(-0.12 -0.02)" fill="#ffbd4f"/><path d="M34.34,69.85,2,39.3A6.4,6.4,0,0,0,.4,42.09a6.17,6.17,0,0,0-.28,1.76v54.3a6.54,6.54,0,0,0,1.95,4.58L32.91,71.3ZM91.05,41.74A6.7,6.7,0,0,0,89.28,39L57.65,69.2l1.43,1.5,30.41,32a6.52,6.52,0,0,0,1.91-4.61V43.75a6.48,6.48,0,0,0-.35-2Z" transform="translate(-0.12 -0.02)" fill="#ffd567"/><path d="M34.34,69.85,45.7,80.68,57.59,69.32,89.21,39.07a6.35,6.35,0,0,0-4.33-1.78H6.68a6.3,6.3,0,0,0-4.6,2Z" transform="translate(-0.12 -0.02)" fill="#ffea80"/><path d="M168.42,101.72l-19.68,18.86L129.8,102.73,127.54,105l-51.2,52.06A10.84,10.84,0,0,0,84,160.31H213.51a10.75,10.75,0,0,0,7.63-3.25l-50.32-52.77Z" transform="translate(-0.12 -0.02)" fill="#e31587"/><path d="M129.8,102.63,76.22,52l.1-.09h-.1a11.22,11.22,0,0,0-3.08,7.53v90a10.77,10.77,0,0,0,3.25,7.63L127.59,105Zm93.92-46.57a11.2,11.2,0,0,0-2.93-4.62h-.09l.09.09-52.37,50.11,2.35,2.47,50.32,52.77a10.75,10.75,0,0,0,3.17-7.63V59.37a10.3,10.3,0,0,0-.54-3.31Z" transform="translate(-0.12 -0.02)" fill="#ff298a"/><path d="M129.8,102.63l18.94,17.86,19.68-18.83,52.37-50.12-.09-.08a10.38,10.38,0,0,0-7.19-3H84a10.55,10.55,0,0,0-7.62,3.34l-.09.1Z" transform="translate(-0.12 -0.02)" fill="#ff9aa2"/><path d="M197.15,104.4a3.6,3.6,0,0,0,.32-.69c0-.94-18.88-32.71-50.75-32.71S96,102.77,96,103.71a3,3,0,0,0,.31.69c2.52-3.51,20.89-23.71,50.41-23.71S194.62,100.89,197.15,104.4Z" transform="translate(-0.12 -0.02)" fill="#c689ff"/><path d="M96.29,104.4a41.56,41.56,0,0,0,2.54,4c6.39,9,23.25,28,47.89,28s41.51-19,47.89-28a41.56,41.56,0,0,0,2.54-4c-2.53-3.51-20.9-23.71-50.43-23.71S98.83,100.89,96.29,104.4Z" transform="translate(-0.12 -0.02)" fill="#d9bfff"/><circle cx="146.6" cy="100.05" r="29.07" fill="#7542e5"/><circle cx="146.48" cy="100.05" r="21.8" fill="#9059ff"/><circle cx="146.66" cy="99.99" r="14.6" fill="#45278d"/><circle cx="155.85" cy="89.79" r="7.84" fill="#fff"/><path d="M194.61,108.45a18.94,18.94,0,0,0,2.86-3.46,2.26,2.26,0,0,0-.32-.54A41.56,41.56,0,0,1,194.61,108.45Z" transform="translate(-0.12 -0.02)" fill="#d9bfff"/></svg>
```

Comment:
Do these SVGs need the Mozilla Privacy license comment? It appears on in the Notification SVG `/public/img/svg/notification.svg`: 

```
<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
```

Reply:
Good question! It doesn't appear on most of our SVGs but I will ask legal. I think we're ok to proceed without it for now and add in later if necessary.